pkg/downloader/downloader.go

```diff
@@ -487,14 +489,11 @@ func downloadHTTP(localPath, url, description string, expectedDigest digest.Dige
 	}
 	defer fileWriter.Close()
 
-	resp, err := http.Get(url)
+	resp, err := httpclientutil.Get(ctx, http.DefaultClient, url)
 	if err != nil {
 		return err
 	}
 	defer resp.Body.Close()
-	if resp.StatusCode != http.StatusOK {
-		return fmt.Errorf("expected HTTP status %d, got %s", http.StatusOK, resp.Status)
-	}
```

Comment:
Why remove?

Reply:
`httpclientutil.Get` already checks for successful status code, see https://github.com/lima-vm/lima/blob/a00bd52f78f86fc611f7c2a098597ae2892762ba/pkg/httpclientutil/httpclientutil.go#L28